src/cmd/linuxkit/cache.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/util"
+	log "github.com/sirupsen/logrus"
+)
+
+func cacheUsage() {
+	invoked := filepath.Base(os.Args[0])
+	fmt.Printf("USAGE: %s cache command [options]\n\n", invoked)
+	fmt.Printf("Supported commands are\n")
+	// Please keep these in alphabetical order
+	fmt.Printf("  clean\n")
+	fmt.Printf("  import\n")
```

Comment:
This doesn't exist?

Reply:
Ah, I never implemented it. Let's leave it out, I will do a separate much smaller PR for it.